bin/pwa.dart

```diff
@@ -232,7 +232,10 @@ class _OfflineUrlScanner {
 
   /// Updates the offline_urls.g.dart file.
   Future writeToFile(String fileName) async {
-    String listItems = offlineUrls.map((s) => '\'$s\',').join();
+    String listItems = offlineUrls
+      .map((s) => '\'$s\',')
+      .map((s) => s.replaceAll('\$', '\\\$'))
```

Comment:
Please add a code comment for the reason of this.

Reply:
Done.